Tasks/NuGetRestoreV1/nugetinstaller.ts

```diff
@@ -255,4 +261,26 @@ function restorePackagesAsync(solutionFile: string, options: RestoreOptions): Q.
     return nugetTool.exec({ cwd: path.dirname(solutionFile) } as IExecOptions);
 }
 
+function _logNuGetRestoreVariables(nuGetPath: string, nuGetVersion: string) {
+    try {
+        const telem = {
+            "solution": tl.getPathInput("solution", true, false),
+            "isNoCacheEnabled": tl.getBoolInput("noCache"),
+            "verbosity": tl.getInput("verbosity"),
+            "packagesDirectory": tl.getPathInput("packagesDirectory"),
+            "NUGET_EXE_TOOL_PATH_ENV_VAR": tl.getVariable(nuGetGetter.NUGET_EXE_TOOL_PATH_ENV_VAR),
+            "nuGetPath": nuGetPath,
+            "nuGetVersion": nuGetVersion,
+            "SYSTEMVSSCONNECTION": tl.getEndpointUrl("SYSTEMVSSCONNECTION", false),
+            "ExtraUrlPrefixesForTesting": tl.getVariable("NuGetTasks.ExtraUrlPrefixesForTesting"),
+            "selectOrConfig": tl.getInput("selectOrConfig"),
+            "nugetConfigPath": tl.getPathInput("nugetConfigPath", false, true),
+            "isIncludeNuGetOrgEnabled": tl.getBoolInput("includeNuGetOrg", false)
+        };
+        telemetry.emitTelemetry("Packaging", "NuGetRestore", telem);
```

Comment:
The task path is ".../nugetinstaller/.." but we're recording CI data under "NuGetRestore". Will that cause confusion?

Reply:
I thought about that and decided it would be confusing no matter what.  Since NuGetRestore using nuGetInstaller under the hood, the Task id's are the same so we had to jump through some silly hoops to identify the actual task invocation in Kusto when gathering metrics.  It seems more intuitive to me that when a dev is looking for CI events, they would search for it based on the actual task name, rather than the task path which may not be totally obvious at first glance.